packages/core/js-peer/src/js-peer/FluencePeer.ts

```diff
@@ -465,7 +462,16 @@ export class FluencePeer implements IFluenceClient {
         this._incomingParticles
             .pipe(
                 tap((x) => {
-                    x.particle.logTo('debug', 'particle received:');
+                    log.trace('id %s. received:', x.particle.id);
+                    log.debug('id %s. data: %j', x.particle.id, {
```

Comment:
```suggestion
                    log.debug('id %s. received:', x.particle.id);
                    log.trace('id %s. data: %j', x.particle.id, {
```
shouldn't it be like that? more info on deeper level (`trace`), less info on higher (`debug`)